frontend/src/components/organisms/AddingPlayerForm/AddingPlayerForm.test.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  render,
+  screen,
+  waitForElementToBeRemoved,
+} from "@testing-library/react";
+import { BrowserRouter as Router } from "react-router-dom";
+import { TournamentRegistrations } from "../../pages/TournamentRegistrations";
+import { PlayerProfileDto } from "../../../restapi/players-profiles";
+import { server } from "../../../mocks/msw/server";
+import { rest } from "msw";
+import userEvent from "@testing-library/user-event";
+
+describe("Adding player form", () => {
+  it("after clicking button form should be shown and display title, inputs and button", async () => {
+    //Given
+    const playersProfiles: PlayerProfileDto[] = [];
+    getPlayersProfilesWillReturn(playersProfiles);
```

Comment:
Co to tutaj robi?

Reply:
musi być, żeby wygenerować widok nawet jeśli przekazujemy pustą listę graczy